pkg/flow/grpc-instance-variables.go

```diff
@@ -298,6 +375,115 @@ func (flow *flow) SetInstanceVariable(ctx context.Context, req *grpc.SetInstance
 
 }
 
+func (internal *internal) SetInstanceVariableParcels(srv grpc.Internal_SetInstanceVariableParcelsServer) error {
+
+	internal.sugar.Debugf("Handling gRPC request: %s", this())
+
+	ctx := srv.Context()
+
+	req, err := srv.Recv()
+	if err != nil {
+		return err
+	}
+
+	inc := internal.db.Instance
+
+	instance := req.GetInstance()
```

Comment:
If these Instance is empty, this might create some hard to debug errors in the future. You might want a `if == ""` check

Reply:
Actually, I notice you call alot of functions that do nil safety checks, but dont check if you get an empty string back.

Reply:
I don't think this matters. The database will return zero results.